tao_compiler/mlir/xla/ral/context/common_context_impl_quantization.cc

```diff
@@ -1376,20 +1407,22 @@ MemRefType<int8_t, 2> ral_pdll_qgemm_onednn_s8_s8_s8_f32_per_channel(
   return result;
 }
 
-MemRefType<int8_t, 2> ral_pdll_qgemm_onednn_s8_s8_s8_per_channel(
+template <int NDims>
+MemRefType<int8_t, NDims> ral_pdll_qgemm_onednn_s8_s8_s8_per_channel(
     ExecutionContext* ctx, opaque_t /*stream_handle*/,
-    MemRefType<int8_t, 2> input, MemRefType<int8_t, 2> weight,
+    MemRefType<int8_t, NDims> input, MemRefType<int8_t, 2> weight,
     MemRefType<float, 0> inputScales, MemRefType<int32_t, 0> inputZeroPoints,
     MemRefType<float, 1> weightScales, MemRefType<int32_t, 1> weightZeroPoints,
     MemRefType<float, 0> resultScales, MemRefType<int32_t, 0> resultZeroPoints,
     void* customAttrs) {
   CpuTimer timer("ral_pdll_qgemm_onednn_s8_s8_s8_per_channel");
-  int64_t resultSizes[2] = {0, 0};
+  int64_t resultSizes[NDims];
```

Comment:
just use `int64_t resultSizes[NDims] = {0};` is ok.

Reply:
done